src/connectors/newgrounds.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const artistSelector = '.ng-apg-media-artist';
+const trackSelector = '.ng-apg-media-title';
+
+function setupConnector() {
+	if (isEmbeddedPlayer()) {
+		setupEmbeddedPlayer();
+
+		Connector.getArtistTrack = () => {
+			let { artist, track } = Util.splitArtistTrack(document.getElementsByTagName('h2')[0].innerText, '-');
+
+			if (!artist) {
+				artist = document.getElementsByTagName('h4')[0].innerText;
+				track = document.getElementsByTagName('h2')[0].innerText;
+			}
+
+			return { artist, track };
+		};
+	} else {
+		setupGlobalPlayer();
+
+		Connector.getArtistTrack = () => {
+			let { artist, track } = Util.splitArtistTrack(Util.getTextFromSelectors(trackSelector), '-');
+
+			if (!artist) {
+				artist = Util.getTextFromSelectors(artistSelector);
+				track = Util.getTextFromSelectors(trackSelector);
+			}
+
+			return { artist, track };
+		};
+	}
+}
+
+function isEmbeddedPlayer() {
+	return $('#audio-listen-player').length > 0;
+}
+
+function setupGlobalPlayer() {
+	Connector.playerSelector = '#_ngHiddenAudioPlayer';
+
+	Connector.playButtonSelector = '#global-audio-player-play';
```

Comment:
You should use either play or pause button selector. Using both selectors is redundant, unless there's a special case to use both (but these cases should be documented).